### ISSUE TITLE
start contribute to the projects

### ISSUE THREAD

**Issue by ohad1992** [cloudevents/sdk-csharp#312]
Hi everyone im new to open source and im willing to start contribute to the project, how can i start ?

**Comment by jskeet** [comment]
You could look through the list of issues, and see if any of them is of interest to you. If one is, then:

- Create a new fork in GitHub
- Fetch your fork locally
- (Optional, but helpful) Create a new branch
- Make the changes you want to see merged
- Put your branch back to GitHub
- Create a pull request from that branch

I don't have time right now to give more detail about all of those steps, but do add a comment if you get stuck (after doing a bit of research).

**Comment by ohad1992** [comment]
ok.
do you a slack or something that i can ask question about the project?

**Comment by jskeet** [comment]
No, I'm afraid not. (I personally find asynchronous communications much more efficient.)

**Comment by ohad1992** [comment]
can we have a zoom meeting?
i want to understand more about the project

**Comment by jskeet** [comment]
No, I'm afraid I don't have time to organise anything like that. Feel free to dump a load of questions here (after researching CloudEvents in general, trying out the SDK etc) and I'll do my best to answer them, but my schedule makes it hard to start arranging synchronous communications.
